AudioBatchMetadata metadata

time start
string[] speech_segments
VoiceActivityMarker voice_markers
